src/test/resources/create-table-with-all-liquibase-types.spanner.yaml

```diff
@@ -31,6 +31,7 @@ databaseChangeLog:
                 type:    BIGINT
                 constraints:
                   primaryKey: true
+                  nullable: false
```

Comment:
Do we still have a test that has primaryKey with nullable: true? Just to ensure that containsNotNullConstraintForColumn etc are all working correctly.

Reply:
Yes, there are a number of them, as every test that does not specify anything will by default be `nullable: true`. But just to be 100% sure I've added the `nullable: true` explicitly to one of the tests as well.